src/parser.rs

```diff
@@ -103,6 +103,7 @@ pub enum SimpleSelector {
     LastOfType,
     OnlyOfType,
     ServoNonzeroBorder,
+    Lang(String),
```

Comment:
This should be an `Atom` rather than `String`, so that comparison can be fast during matching.